openfunction/values.yaml

```diff
@@ -0,0 +1,77 @@
+    # W3C Baggage Specification/: https://w3c.github.io/baggage/
+      key: sw8-correlation # key should be baggage for opentelemetry
+      value: "base64(string key):base64(string value),base64(string key2):base64(string value2)"
+controllerManager:
+  kubeRbacProxy:
+    image:
+      repository: openfunction/kube-rbac-proxy
+      tag: v0.8.0
+  openfunction:
+    image:
+      repository: openfunction/openfunction
+      tag: latest
+    resources:
+      limits:
+        cpu: 500m
+        memory: 500Mi
+      requests:
+        cpu: 100m
+        memory: 20Mi
+  replicas: 1
+kubernetesClusterDomain: cluster.local
+managerConfig:
+  controllerManagerConfigYaml:
+    health:
+      healthProbeBindAddress: :8081
+    leaderElection:
+      leaderElect: true
+      resourceName: 79f0111e.openfunction.io
+    metrics:
+      bindAddress: 127.0.0.1:8080
+    webhook:
+      port: 9443
+metricsService:
+  ports:
+    - name: https
+      port: 8443
+      targetPort: https
+  type: ClusterIP
+webhookServerCert:
+  tlsCrt: "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUQvVENDQXVXZ0F3SUJBZ0lVQnV6QmJ6WDloSHlNL0tCNFpveFBhRmxkNWlvd0RRWUpLb1pJaHZjTkFRRUwKQlFBd0tqRW9NQ1lHQTFVRUF3d2ZZMkV0YjNCbGJtWjFibU4wYVc5dUxYZGxZbWh2YjJzdGMyVnlkbWxqWlRBZQpGdzB5TWpBME1EY3dNelV3TURaYUZ3MHpNakEwTURRd016VXdNRFphTUNjeEpUQWpCZ05WQkFNTUhHOXdaVzVtCmRXNWpkR2x2YmkxM1pXSm9iMjlyTFhObGNuWnBZMlV3Z2dFaU1BMEdDU3FHU0liM0RRRUJBUVVBQTRJQkR3QXcKZ2dFS0FvSUJBUURIK3BWZ1VKajJqbnNuZE03ay9VVFJwaWUwZHdWSHNYV2VOV0hwZEtQN2NXZVZqVlRLZHB5OApQdjBaQ3FjTlppQXlTUTNMcWJ0ejNLc0VUR3pxT091cDRyWUhTSDRMamJzSUtObGVZN0lmQkVkdFZ1Nlc0NjBPClkySVEvYmNWNU00R2xOb0RNdXd0azNHWGlsK3hNQkx0NUNFeHg5VGNXTmxrZzFrOFFlYWh6UUpxakJwNTFkTU8KSXprQVF4SGdmaUV3WXZPdjg4SmxGdDJvQkh1RHpCYWVzRnFlRTA3ck5BT3NnQmgvNEJXMmxva1pkODY3YnZPSgp4MjUybk9CbWxtWEFieG9ldGtHRlEzZnM4b0VaeWVCS0haWXhVYmVxSG9xaUlxaGFrZEUyajFjTVc4S1FEK3ZSCmFFMnZKMWlEc2lHQjQ5cENtMGhDL0lHeFVwM2I3SFBOQWdNQkFBR2pnZ0VjTUlJQkdEQUpCZ05WSFJNRUFqQUEKTUFzR0ExVWREd1FFQXdJRjREQWRCZ05WSFNVRUZqQVVCZ2dyQmdFRkJRY0RBZ1lJS3dZQkJRVUhBd0V3Z2Q0RwpBMVVkRVFTQjFqQ0IwNEljYjNCbGJtWjFibU4wYVc5dUxYZGxZbWh2YjJzdGMyVnlkbWxqWllJcGIzQmxibVoxCmJtTjBhVzl1TFhkbFltaHZiMnN0YzJWeWRtbGpaUzV2Y0dWdVpuVnVZM1JwYjI2Q0xXOXdaVzVtZFc1amRHbHYKYmkxM1pXSm9iMjlyTFhObGNuWnBZMlV1YjNCbGJtWjFibU4wYVc5dUxuTjJZNEk3YjNCbGJtWjFibU4wYVc5dQpMWGRsWW1odmIyc3RjMlZ5ZG1salpTNXZjR1Z1Wm5WdVkzUnBiMjR1YzNaakxtTnNkWE4wWlhJdWJHOWpZV3lDCkhHOXdaVzVtZFc1amRHbHZiaTEzWldKb2IyOXJMWE5sY25acFkyVXdEUVlKS29aSWh2Y05BUUVMQlFBRGdnRUIKQUkxQnVhSlM5WUxHOUlYSU0rSUVYU3p5bW80N3BpVzkrQS8wQUtrK0RnVUpPMEZURDJqZEtQNkZ5WTB2VDJxRQo5OTNxZE5XZXdSdHQ4N1ZxUmQvVHNRQWVnMHhZZWxURUVOb0RBZjYzSEIzWEY2b0JmUHRkaU1XMGgxSnYwendsCjdRN2cxZTZNVDBaN0Zxb21ZQ1VFOC8zTG1oMHhWcTNHL1cvS0Fnd1JCQkZlekl4WlM5NVg3UkJsVkdoSkI5WnoKTmY3cmtEVDhwOFpnZ3JjZUE4RHNYM3lPMEF4cGVuS0RVRUNQR1doQVVESG1pVmFURURoZU1Vd3pyZCtIWUdqYwpnenR2Y0RXMGJSWkVJWDJsQUZDWHFESUhPNGRiODNxT2FFdHh2SlhyY1NvdEgvUnVmV1p0RWc0bjhqN0E0L0c5CjVqWjhVSTE4bXFoM1RmNnpzd0lhQUk4PQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg=="
```

Comment:
This certificate is only valid when openfunction is deployed in `openfunction` namespace, how to deploy openfunction to another namespace?

Reply:
The readme.md documentation already states `kubectl create ns openfunction` before install. helm automatically generates the certificate and modifies the corresponding crd and template files is a better way, but there are some issues.

Reply:
Support deploying to different ns is good, but it's not as urgent as the notification manager.
Notification manager is integrating with kubesphere and is installed to kubesphere-monitoring-system.
But users can also use it without kubesphere and then it might be installed to another different ns.

Installing OpenFunction to a different ns than openfunction is not as urgent as nm.
 
But we can create an issue if it's not included in this PR @wrongerror

Reply:
I will create an issue about installing to different namespace. The reason for the problem is that the CRD requires the caBundle field for conversion, but crd itself does not support being rendered by the template. I've discussed this a lot with @tpiperatgod, I'm inclined to create a `pre-install` job to patch, if needed I think it can be done soon.

Reply:
You can take a look at how wanjun solved a problem like this: https://github.com/kubesphere/notification-manager/pull/141